client/packages/system/src/Name/DetailModal/DetailModal.tsx

```diff
@@ -114,14 +106,54 @@ export const DetailModal: FC<DetailModalProps> = ({ nameId }) => {
             <DetailInputWithLabelRow
               label={t('label.address')}
               inputProps={{
-                value: [data?.address1, data?.address2]
-                  .filter(a => !!a)
-                  .join(', '),
+                value: data?.address1,
                 disabled: isDisabled,
                 maxRows: 3,
                 multiline: true,
               }}
             />
+            <DetailInputWithLabelRow
+              label={t('label.address')}
+              inputProps={{
+                value: data?.address2,
+                disabled: isDisabled,
+                maxRows: 3,
+                multiline: true,
+              }}
+            />
```

Comment:
Since this is display only, I wonder if we should put both address components into the one field? It seems a bit weird to me to have two fields labelled "Address". Could they be two lines in a single display?